HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -3183,7 +3193,22 @@ namespace WPEFramework
            _instance->smConnection->sendTo(LogicalAddress::AUDIO_SYSTEM,MessageEncoder().encode(ReportArcTermination()), 1000);
 
        }
-
+       
+      void HdmiCecSink::getHdmiArcPortID()
```

Comment:
Do we need a separate API?  Can we directly call void AudioOutputPort::getHdmiArcPortId(int *portId)  and initialize HdmiArcPortID